src/server/game/Entities/Object/Object.h

```diff
@@ -452,12 +452,22 @@ class FlaggedValuesArray32
         T_FLAGS m_flags;
 };
 
+enum class FindCreatureAliveState : uint8
+{
+    Alive               = 0, // includes feign death
+    Dead                = 1, // includes feign death
+    EffectivelyAlive    = 2, // excludes feign death
+    EffectivelyDead     = 3, // excludes feign death
```

Comment:
I would have expected "EffectivelyDead" to mean "behaves like dead", which means `!IsAlive() || IsFeignDeath()`